common/views/components/PrismicImage/PrismicImage.tsx

```diff
@@ -21,6 +21,7 @@ export type BreakpointSizes = Partial<Record<Breakpoint, number>>;
 export type Props = {
   image: ImageType;
   sizes?: BreakpointSizes;
+  imgSizes?: string;
```

Comment:
I've created another prop here to give us more control over the `sizes` attribute if necessary. The existing `sizes` prop is a bit of a blunt instrument and doesn't account for different amounts of images per column at different screen sizes.